src/main/java/io/confluent/examples/streams/TopArticlesLambdaExample.java

```diff
@@ -142,7 +142,6 @@ static KafkaStreams buildTopArticlesStream(final String bootstrapServers,
     // Specify default (de)serializers for record keys and for record values.
     streamsConfiguration.put(StreamsConfig.DEFAULT_KEY_SERDE_CLASS_CONFIG, Serdes.String().getClass().getName());
     streamsConfiguration.put(StreamsConfig.DEFAULT_VALUE_SERDE_CLASS_CONFIG, GenericAvroSerde.class);
-    streamsConfiguration.put(StreamsConfig.COMMIT_INTERVAL_MS_CONFIG, 1000);
```

Comment:
more out of curiosity, why this change?

Reply:
Because commit interval is set to 10 seconds in line 149.